tests/specs/scoped.js

```diff
@@ -592,7 +592,7 @@ test('scoped Store .off()', function (t) {
   })
 })
 
-test('when type change', function (t) {
+test.only('when type change', function (t) {
```

Comment:
standard is nice but it just covers the basics of linting style and not anything specific to your own project/framework/etc (if you want to take it farther).